### DISCUSSION TITLE
Always Remember User option

### DISCUSSION THREAD

**Discussion by enfJoao** [oqtane/oqtane.framework#4840] [General]
From a security perspective, this option could be tweaked to "Remember User" with the option to "Automatic", "Optional" and "Disabled".

Automatic would match the current always remember.
Optional the show checkbox for user to tick.
Disabled to never alow the persistent login. Every project that depends on sensitive data access benefits from 2FA, short timed logging and never keeping logins active after closing tabs/browsers.

**Comment by sbwalker** [comment]
@enfJoao you are referring to an enhancement to the User Settings:

![image](https://github.com/user-attachments/assets/c5e36f52-85d0-4ce4-93d8-7ebc51e6ff53)

Currently when you set the option to Yes it displays the checkbox on the Login screen and allows the user to make a selection - which is how you described Optional above. I am not sure I understand what Automatic would do.

**Comment by enfJoao** [comment]
The options that exist are fine.
It's just missing the option to disable the "Remember me".

![image](https://github.com/user-attachments/assets/276fc4f7-327f-44df-aec3-ba618d3f7962)

If a company stores sensitive data, the remember me option is a security risk.
